scripts/count-assertions.js

```diff
@@ -0,0 +1,163 @@
+const path = require('path');
+const fs = require('fs');
+
+const csv = require('../lib/util/csv');
+
+const args = require('minimist')(process.argv.slice(2), {
+  alias: {
+    h: 'help',
+  },
+});
+
+if (args.help) {
+  console.log(`
+Default use:
+  node count-assertions.js [directory]
+    Counts test plans, tests, assertions, and command/assertion pairs
+    for all directories. Counts only a single directory if specified as
+    optional first argument.
+  Arguments:
+    -h, --help
+      Show this message.
+`);
+  process.exit();
+}
+
+const testsDirectory = path.resolve(__dirname, '..', 'tests');
+
+/**
+ * @param {string} filepath
+ * @returns {Promise<object[]>}
+ */
+function parseCSV(filepath) {
+  return csv.read(fs.createReadStream(filepath), {
+    logError: (...args) => console.error(`[${path.relative(testsDirectory, filepath)}]`, ...args),
+  });
+}
+
+/**
+ * Count metrics for arrays of test and command objects.
+ * @param {AriaATCSV.Test[]} tests
+ * @param {AriaATCSV.Command[]} commands
+ */
+function countTests(tests, commands) {
+  let totalTests = 0;
+  let totalAssertions = 0;
+  let totalCommandAssertions = 0;
+
+  for (const [i, test] of tests.entries()) {
+    const command = commands[i];
+
+    let numCommands = 0;
+    for (const letter of ['A', 'B', 'C', 'D', 'E', 'F']) {
```

Comment:
Since merging #832, `commandG - commandZ` (up to 26 commands) are supported and at least 2 tests at this time support up to 7 commands (`commandG`).

Will push a commit to update this.

Reply:
See https://github.com/w3c/aria-at/pull/442/commits/aaaeb6f7714fc6ba276f6c1c397ffb4f4769afdb.